src/main/java/nl/tudelft/pixelperfect/gamestates/PlayState.java

```diff
@@ -118,4 +118,9 @@ public GameState handleState() {
     }
     return this;
   }
+
+  @Override
+  public boolean isRunning() {
+    return true;
+  }
```

Comment:
Yes, but in practice this will never happen. The purpose of the states is self-evident, they describe the "state" the system is in. If the above scenario is able to occur, then we are not using the state design pattern correctly, and the error is not in this usage, but the implementation of the state transitions.